HACKING.md

```diff
@@ -92,4 +92,58 @@ To update the blackbox compiler tests set TeamCity build number in `gradle.prope
 
 * **-Ptest_verbose** enables printing compiler args and other helpful information during a test execution.
 
-        ./gradlew -Ptest_verbose :backend.native:tests:mpp_optional_expectation
\ No newline at end of file
+        ./gradlew -Ptest_verbose :backend.native:tests:mpp_optional_expectation
+       
+ ## Performance measurement
+  
+ Firstly, it's necessary to build analyzer tool to have opportunity to compare different performance results:
+ 
+    cd tools/benchmarksAnalyzer
+    ../../gradlew build
+    
+ To measure performance of Kotlin/Native compiler on existing benchmarks:
+ 
+    cd performance
+    ../gradlew konanRun
+    
+ **konanRun** task can be run separately for one/several benchmark applications:
+ 
+    ../gradlew :cinterop:konanRun
+    
+ **konanRun** task has parameter `filter` which allows to run only some subset of benchmarks :
+ 
+    ../gradlew :cinterop:konanRun --filter=struct,macros
+    
+ There are also tasks for running benchmarks on JVM (pay attention, some benchmarks e.g. cinterop benchmarks can't be run on JVM)
```

Comment:
BTW, why cannot we run interop benches on JVM? It mostly works.

Reply:
I used functions from kotlin/native stdlib. To try run them on JVM I need to write realization for JVM. But I don't think that it's very useful, because it can't be really compared.